tools/docker/Dockerfile.tdx

```diff
@@ -0,0 +1,166 @@
+# SPDX-License-Identifier: MPL-2.0
+
+FROM intelcczoo/tdvm:ubuntu22.04-mvp_2023ww15 as build-base
+
+SHELL ["/bin/bash", "-c"]
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+USER root
+
+ENV http_proxy=
+ENV https_proxy=
+
+# Please keep the list sorted by name
+RUN apt update && apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    git-core \
+    gnupg \
+    libssl-dev \
+    python3-pip \
+    python-is-python3 \
+    wget
+
+#= Build benchmark =========================================================
+
+FROM build-base as build-benchmarks
```

Comment:
If possible, busybox, benchmarks, and gvisor test can be copied from the Asterinas image to avoid compiling them again.

Reply:
Do you mean copying files from a Asterinas Docker image layer?  For example:

```
FROM asterinas:tag as intermediate

FROM tdx:tag
COPY --from=intermediate /path/in/asterinas/image /path/in/tdx/image
```

Or copying a layer from the Asterinas Docker image?

Reply:
Seems inconvenient to implement.